assets/css/nojs.css

```diff
@@ -0,0 +1,16 @@
+#noscript {
+    position: fixed;
+    right: 0;
+    text-align: center;
+    top: 5px;
+    width: 100%;
+    z-index: 10000000000000;
+}
+
+.navbar {
+    margin-top: 4em;
+}
+
+body > .container {
+    margin-top: 4em;
+}
```

Comment:
I think these can all just go in `general` :)

Reply:
So all of `nojs` into `general` except the rule-set for `.navbar`?